official/vision/beta/projects/yolo/modeling/layers/nn_blocks_test.py

```diff
@@ -281,5 +283,102 @@ def test_gradient_pass_though(self, filters, width, height, downsample):
 
     self.assertNotIn(None, grad)
 
+
+class DarkSppTest(tf.test.TestCase, parameterized.TestCase):
```

Comment:
Not sure about the abbreviation Spp here

Reply:
Abbreviation for [spatial pyramid pooling](https://arxiv.org/abs/1406.4729)